src/styles/queryInput.scss

```diff
@@ -0,0 +1,8 @@
+.whppt-query-input {
+  @apply relative w-full flex-1;
+
+  & button {
+    @apply absolute right-3 p-4 h-4 text-base;
+    top: 10px;
```

Comment:
Can use tailwind top-2 or top-3.
I prefer to use the tailwind classes here because we can tell the designers to stick to the tailwind docs.